### PR TITLE
Sorting by GitHubUpdatedAt field

### LINKED ISSUE #5: Fix issue sorting
Each github issue has an updatedAt field. That field should be stored on a new database column. Once this is done the issue listing should sort the issues based on that column

### LINKED ISSUE #12: Finish up work started at #7
#7 includes some core functionality that will allow us to improve sorting of issues. The pull request is almost done but it needs a few improvements that are listed inside the pull request

### PR DESCRIPTION
**Pull request by himankpathak**
### Description of the Change
Added GitHubUpdatedAt field which is fetched from GitHub's GraphQL API "updatedAt". It is stored in database at column GitHubUpdatedAt.

### Benefits
Now issues can be sorted using GitHubUpdatedAt field.

### Applicable Issues
Closes #5 
Closes #7
Closes #12

### CODE REVIEW

**Comment by himankpathak** [comment]
@alexdor should a new issue be opened in the [frontend](https://github.com/ossn/fixme/) to sort using GitHubUpdatedAt field?

File: models/issue.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ type Issue struct {
 	ID               uuid.UUID     `json:"id" db:"id"`
 	CreatedAt        time.Time     `json:"created_at" db:"created_at"`
 	UpdatedAt        time.Time     `json:"updated_at" db:"updated_at"`
+	GithubUpdatedAt  time.time		 `json:"github_updated_at" db:"github_updated_at"`
```
Comment (alexdor) [inline models/issue.go]: Could you fix the formatting here? You can use go fit to fix all the formatting issues

**Review by alexdor — CHANGES_REQUESTED** [review]
Looking good : ) Could you also set the GithubUpdateAt field as the default sorting for queries?

**Comment by alexdor** [comment]
> @alexdor should a new issue be opened in the [frontend](https://github.com/ossn/fixme/) to sort using GitHubUpdatedAt field?

Yes, once this is merged we should do that : )

File: models/issue.go (hunk 1 above)
Comment (himankpathak) [reply models/issue.go]: Where should I set the default sorting for queries?

Comment (alexdor) [reply models/issue.go]: Hm, I don't find it in the buffalo docs, maybe they haven't implemented this feature yet. You can update this query https://github.com/ossn/fixme_backend/blob/master/actions/issues.go#L41 to order the issues based on the GithubUpdateAt

Comment (himankpathak) [reply models/issue.go]: Hi @alexdor, I cant find the syntax for orderby query over [here](https://gobuffalo.io/en/docs/db/querying/) 
Can you please guide me on how to do it?

Comment (alexdor) [reply models/issue.go]: Unfortunately the documentation isn't complete. Try adding `.Order("github_updated_at desc")` to the query

Comment (alexdor) [reply models/issue.go]: @himankpathak did the above help?

Comment (himankpathak) [reply models/issue.go]: yes it worked

**Comment by alexdor** [comment]
@himankpathak this still has conflicts, could you resolve the conflicts?

**Comment by himankpathak** [comment]
I think I resolved all the conflicts with git rebase. The PR says "This branch has no conflicts with the base branch".
Am I missing something?

**Comment by alexdor** [comment]
I'm getting this error so I don't think they where solved, you can try creating a new fork and cherry-picking your commits to that one
Screenshot 2019-10-06 at 22 57 20

**Comment by himankpathak** [comment]
This is weird, on my side I don't know why it's not showing the conflicts.

Screenshot from 2019-10-07 09-39-23

Should I close this PR and make a copy of this branch from commit https://github.com/ossn/fixme_backend/pull/17/commits/cd0f7ef96eaa9efc43ebe527137b34aa25f48b3c and add commit https://github.com/ossn/fixme_backend/pull/17/commits/d6f1492109c6e48f173d36568fe14ff91f59fe36?

**Comment by alexdor** [comment]
Yeah, try creating a new pr, maybe that will solve the issue. Before you copy the branch, try just closing this pr and opening a new one, maybe it’s a GitHub issue that it shows conflicts

Best,
Alex

**Pull request closed without merging**